canvas_modules/harness/assets/styles/common.scss

```diff
@@ -14,14 +14,5 @@
  * limitations under the License.
  */
 $font-prefix: "/fonts";
-@import "@ibm/plex/scss/ibm-plex.scss";
-
-@import "@carbon/themes/scss/themes";
-
-// Switch the themes below to get light or dark.
-
-$carbon--theme: $carbon--theme--g10; // The light (gray 10) theme
-// $carbon--theme: $carbon--theme--g90; // The dark theme
-@include carbon--theme;
-
-@import "carbon-components/scss/globals/scss/styles";
\ No newline at end of file
+@use "@carbon/react" as *;
+@import "node_modules/@ibm/plex/scss/ibm-plex.scss";
```

Comment:
Do we even need this file anymore?  This was originally intended to contain only the carbon styling and in our code we'd only pull in the variables.